README.md

```diff
@@ -1,4 +1,4 @@
-# [jade-data-repository](https://data.terra.bio/) &middot; [![GitHub license](https://img.shields.io/github/license/DataBiosphere/jade-data-repo)](https://github.com/DataBiosphere/jade-data-repo/blob/develop/LICENSE.md) [![Integration test and Connected tests](https://github.com/DataBiosphere/jade-data-repo/workflows/Integration%20test%20and%20Connected%20tests/badge.svg?branch=develop)](https://github.com/DataBiosphere/jade-data-repo/actions?query=workflow%3A%22Integration+test+and+Connected+tests%22+branch%3Adevelop)
+# [jade-data-repository](https://data.terra.bio/) &middot; [![GitHub license](https://img.shields.io/github/license/DataBiosphere/jade-data-repo)](https://github.com/DataBiosphere/jade-data-repo/blob/develop/LICENSE.md) [![Unit, Smoke, Connected and Integration tests](https://github.com/DataBiosphere/jade-data-repo/actions/workflows/int-and-connected-test-run.yml/badge.svg?branch=develop)](https://github.com/DataBiosphere/jade-data-repo/actions/workflows/int-and-connected-test-run.yml?query=branch%3Adevelop)
```

Comment:
I noticed the build badge was broken, I think it was broken by the change that renamed the workflow https://github.com/DataBiosphere/jade-data-repo/pull/1349